### DISCUSSION TITLE
[Bug]: Editable prints HTML markup

### DISCUSSION THREAD

**Discussion by zangab** [pimcore/pimcore#9990] [Q&A / Support] [answered]
### Expected behavior

As seen at the pimcore docs for the [areabrick blocks](https://pimcore.com/docs/pimcore/current/Development_Documentation/Documents/Editables/Areablock/Bricks.html#page_How-to-Create-a-Brick) I created my `view.html.twig` files for the bricks with all fields defined atop. I really like the approach as it makes everything cleaner. However, if I render the fields, the markup will be printed. This results in requiring the raw filter `textField | raw`. I don't think this is the way-to-go or intended as the docs samples show otherwise??

![block-editable-output](https://user-images.githubusercontent.com/9076592/128832868-e301b0e7-40ce-4ae1-b100-7646e01b4f57.png)

### Actual behavior

No need for the `raw` filter to render an editable field in editmode. 

### Steps to reproduce

```twig
{% set textField = pimcore_input('infoline', { 'required': true, 'placeholder': 'Lorem ipsum....' }) %}

<aside class="my-super-class" style="padding: 2rem 3rem;">
  {# ends up rendering markup, but no input #}
  {{ textField }}
</aside>
```

**Comment by dvesh3** [comment, marked as answer]
By default, Twig auto escapes the content to protect from XSS attacks. 

So either you mark data as safe by using `raw` filter or wrap your content in `autoescape false` like:
```twig
{% autoescape false %}
...
{% endautoescape %}
```

**Reply by dvesh3** [reply]
Already mentioned in docs https://pimcore.com/docs/pimcore/current/Development_Documentation/MVC/Template/index.html#page_Pimcore-Editables

**Reply by zangab** [reply]
thx for clarification.
neither `autoescape` nor `raw` is used in the areabrick demo though so it's quite confusing. pimcore could have handled that internally. that plain (text-) info ... well, could be done better tbh.
